spec/models/news_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe News, type: :model do
+  let!(:news) { create(:news) }
+
+  it 'has a valid factory' do
+    expect(build(:category)).to be_valid
+  end
+
+  describe 'validations' do
+    it { is_expected.to validate_presence_of(:name) }
+    it { is_expected.to validate_presence_of(:content) }
+    it { is_expected.to validate_attachment_presence(:image) }
+  end
+
+  describe 'associations' do
+    it { should belong_to(:category).required }
```

Comment:
Rubocop recommends the use of `is_expected.to` over `should`. Please check the offenses with `bundle exec rubocop` in your terminal.